packages/backend/src/open_payments/payment_pointer/model.test.ts

```diff
@@ -353,4 +366,63 @@ export const getRouteTests = <M extends PaymentPointerSubresource>({
   }
 }
 
-test.todo('test suite must contain at least one test')
+describe('Payment Pointer Model', (): void => {
```

Comment:
I wasn't completely sure what was going on in this existing test file. It looks like this file just defines and exports some test infrastructure that is used in `open_payments/incoming`, 
 ` open_payments/outgoing`, `open_payments/quote` (hence the original test.todo). 
 
I don't necessarily think there is a problem here but wanted to note that.